CONTRIBUTING.md

```diff
@@ -8,6 +10,28 @@
 6. Push to the branch (`git push origin feat/new-feature`)
 7. Open a pull request
 
+## Working with Docker
+
+This project provides a Docker setup that allows working on it using any of the supported PHP versions.
+
+To use it, you first need to install:
+
+* [Docker](https://docs.docker.com/get-docker/)
+* [Docker Compose](https://docs.docker.com/compose/install/)
+
+Make sure the versions installed support [Compose file format 3.9](https://docs.docker.com/compose/compose-file/).
+
+You can then build the images:
+
+```console
+docker-compose build --parallel
+```
+
+Now you can run commands needed to work on the project. For example, say you want to install the dependencies on PHP 8.0:
+
+```console
+docker-compose run php-80 composer install
+```
```

Comment:
we wolud to ad also che example to enter in container?

Reply:
Updated @Trusted97